crates/gossip/src/gossip.rs

```diff
@@ -244,7 +244,10 @@ impl K2Gossip {
             .peer_meta_store
             .new_ops_bookmark(target_peer_url.clone())
             .await?
-            .unwrap_or(UNIX_TIMESTAMP);
+            .unwrap_or_else(|| {
```

Comment:
I was setting a 0 time for this when I was thinking we'd be incrementally syncing on first join but that's not right. We want to just request what's new that the DHT diff won't catch. We also reset this value if it's been a while since we gossiped with a peer so we really don't want to jump back to timestamp 0.

Reply:
Yes, fair. Extracted a common function and an explanation comment